radio/src/targets/taranis/hal.h

```diff
@@ -2025,7 +2037,7 @@
 #define KEYS_BACKLIGHT_RCC_AHB1Periph        0
 
 // LCD driver
-#if defined(RADIO_TX12) || defined(RADIO_ZORRO) || defined(RADIO_FAMILY_JUMPER_T12) || defined(RADIO_T8) || defined(RADIO_TPRO)
+#if defined(RADIO_TX12) || defined(RADIO_ZORRO) || defined(RADIO_FAMILY_JUMPER_T12) || defined(RADIO_T8) || defined(RADIO_TPRO) || defined(RADIO_LR3PRO)
```

Comment:
You defined earlier `RADIO_FAMILY_JUMPER_T12`, so why do you need the extra `|| defined(....)`?

Reply:
In this way, it seems that their functions are the same. I can delete the redundant macro definitions.